site/src/modules/dashboard/Navbar/DeploymentDropdown.tsx

```diff
@@ -149,7 +149,7 @@ const styles = {
 	menuItem: (theme) => css`
     text-decoration: none;
     color: inherit;
-    gap: 20px;
+    gap: 8px;
```

Comment:
This was a change from when the feature badges were still part of the dropdown. Even with that removed, though, the gap seemed way too big for any feature elements we add